earth_enterprise/src/fusion/autoingest/StorageManager.h

```diff
@@ -21,11 +21,16 @@
 #include <string>
 #include <time.h>
 #include <vector>
+#include <memory>
 
 #include "common/khCache.h"
 #include "common/khFileUtils.h"
 #include "common/notify.h"
 #include "common/SharedString.h"
+#include "common/khxml/khxml.h"
+#include "common/khxml/khdom.h"
+
+using namespace khxml;
```

Comment:
Probably ought to pull up the include below this so that it's above the `using` statement.

Reply:
It got moved out to AssetSerializer.h anyway.